spec/mailers/authentication_mailer_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe AuthenticationMailer, type: :mailer do
+  subject(:mailer) { described_class }
+
+  describe 'send sign_up email' do
```

Comment:
This is inconsistent with the method name (but see comment on the `Mailer` itself re the method name)